Model/Validation/PreventAdminActions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Paytrail\PaymentService\Model\Validation;
+
+use Magento\Customer\Model\Session;
+
+class PreventAdminActions
+{
+    const KEY_LOGIN_AS_CUSTOMER_SESSION = 'logged_as_customer_admind_id';
```

Comment:
admind_  was this a typo?
Expected admin_

Reply:
<img width="316" alt="image" src="https://user-images.githubusercontent.com/85225665/199469943-87af44dd-2d28-46c9-a564-b7ef84ea8bbc.png">
That's correct key